src/pages/SignIn.js

```diff
@@ -1,37 +1,16 @@
 import { Component, Fragment } from 'react'
 import { div, h, p } from 'react-hyperscript-helpers'
-import { Clickable, HeroWrapper, Link } from 'src/components/common'
-import Modal from 'src/components/Modal'
+import { HeroWrapper, Link } from 'src/components/common'
 import SignInButton from 'src/components/SignInButton'
 import colors from 'src/libs/colors'
 import { isBioDataCatalyst, isFirecloud } from 'src/libs/config'
 import { getAppName } from 'src/libs/logos'
-import * as Nav from 'src/libs/nav'
 import * as Style from 'src/libs/style'
 import * as Utils from 'src/libs/utils'
 
 
-export const CookiesModal = (({ onDismiss }) => {
-  return h(Modal, {
-    showCancel: false,
-    onDismiss
-  }, [
-    `${getAppName()} uses cookies to enable sign on and other essential features when signed in, and to provide statistics to our development team regarding how the site is used. For more information, see our `,
-    h(Link, {
-      ...Utils.newTabLinkProps,
-      href: Nav.getLink('privacy')
-    }, ['privacy policy.'])
-  ])
-})
-
 export default class SignIn extends Component {
```

Comment:
Since this no longer has any state, you might take this opportunity to convert it to a function component.

Reply:
Yeah, sure! Will do.